kiwi/xml_state.py

```diff
@@ -470,6 +470,18 @@ def get_build_type_vmdvd_section(self):
             if vmdvd_sections:
                 return vmdvd_sections[0]
 
+    def get_build_type_vmconfig_entries(self):
```

Comment:
I think get_build_type_vmconfig_entries returns None if the whole <machine> section is missing, in that case xml_parse::get_vmconfig_entry() is not called and then in vmdk.py custom_entries is None. The current description files I am using to build terraform images fails with that code, as they don't have <machine> section.